src/locales/en.json

```diff
@@ -0,0 +1,149 @@
+{
+  "error": {
+    "trello": "Trello failed: "
+  },
+  "api": {
+    "add": "Card created",
+    "form_step_two_exception": "List or Board not provided",
+    "configure": {
+      "config_failed": "Unable to submit configuration form: Trello failed ",
+      "config_success": "Successfully updated Trello configuration"
+    },
+    "connect": {
+      "save": "Trello account successfully connected!",
+      "disconnect_exception": "You are not connect to a Trello account.",
+      "disconnect_success": "Trello account successfully disconnected!"
+    },
+    "help": {
+      "header": "Mattermost Trello Plugin - Slash Command Help",
+      "command_help": "Launch the Trello plugin command line help syntax, check out the [documentation]({{ homepageUrl }}).",
+      "command_configure": "Configure Trello workspace",
+      "command_create_description": "Create a new card",
+      "command_add_subcription": "Subscribe channel to receive notifications from a Trello board",
+      "command_list_subcription": "Show all your current subscriptions",
+      "command_stop_subcription": "Stop receiving Trello board's notifications",
```

Comment:
```suggestion
      "command_stop_subcription": "Stop receiving notifications for a Trello board",
```

Reply:
I'm not 100% certain of context here to know whether "a Trello board" or "this Trello board" would be more appropriate. If the action being taken is happening on the _current_ board, please update such strings to "this" rather than "a" consistently.

If the action being taken isn't within the context of a specific board, "a" is appropriate instead.